controllers/session.go

```diff
@@ -65,3 +74,30 @@ func (c *SessionController) Create() {
 	flash.Store(&c.Controller)
 	c.Redirect(redirectPath, http.StatusFound)
 }
+
+// Delete handles revoke a session for an authenticated user
+// @Title Delete
+// @Description revoke a session
+// @Success 302 redirect to root path with success message
+// @Failure 302 redirect to current path with error message
+// @router / [get]
+func (c *SessionController) Delete() {
+	flash := web.NewFlash()
+	redirectPath := "/"
+
+	u, err := url.Parse(c.Ctx.Request.Header.Get("Referer"))
```

Comment:
@Lahphim Can you point me out what is this for?

Reply:
@carryall I just want to improve a little bit with user experience when signing out failed by redirect to the previous path.

For example, I'm in the dashboard page then press sign out and get an error after that somehow.
I should stay at the dashboard page.

`Referer` is auto attached to header to tell us that where we come from.